bagit.xml

```diff
@@ -228,66 +231,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 document are to be interpreted as described in <xref target="RFC2119"/>.
 </t>
 
-<t>
-An implementation is not compliant if it fails to satisfy one or
-more of the MUST or REQUIRED level requirements for the protocols
-it implements.
-</t>
 </section> <!-- /Requirements -->
 
 <section title="Terminology">
 <t>
 This specification uses a number of terms to describe BagIt, some
 of which are in common use, some of which are newly defined by this
-specification, and others which may have meanings obvious only
-to those in the community from which this spec arose. Terms defined
-in this section are intended to clarify any ambiguity.
+specification. Terms defined in this section are intended to clarify 
+any ambiguity.
 </t>
 
 <t>
 <list style="hanging">
 	<t hangText="bag">
-	A set of opaque data contained within the structure defined
+	A order independent set of opaque files contained within the structure defined
 	by this specification.
 	</t>
 
 	<t hangText="bag declaration">
-	The tag file required to be in all bags conforming to this
-	specification. Contains tags necessary for bootstrapping the
+	The file required to be in all bags conforming to this
+	specification. Contains values necessary for bootstrapping the
```

Comment:
“Contains values necessary for bootstrapping the reading and processing of the rest of a bag” could be simplified to something like “Contains values necessary to read the bag”.